app/src/ui/remove-repository/confirm-remove-repository.tsx

```diff
@@ -49,8 +49,8 @@ export class ConfirmRemoveRepository extends React.Component<
         </DialogContent>
         <DialogFooter>
           <ButtonGroup destructive={true}>
-            <Button type="submit">Cancel</Button>
-            <Button onClick={this.onConfirmed}>Remove</Button>
+            <Button type="submit">Remove</Button>
+            <Button onClick={this.onConfirmed}>Cancel</Button>
```

Comment:
:arrow_up: Same thing here - `Dialog.onSubmit={this.onConfirmed}`

Reply:
Same here, `<Button onClick={this.onConfirmed}>` is gonna be removed?